exercises/bob/.meta/generator/test_template.erb

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+require 'minitest/autorun'
+require_relative '<%= exercise_name %>'
+
+# Common test data version: <%= canonical_data_version %> <%= abbreviated_commit_hash %>
+class <%= exercise_name_camel %>Test < Minitest::Test
+  def bob
+    Bob
+  end
```

Comment:
I honestly just tried to reproduce the test pretty much in it's entirety. I agree this method is especially useless. I'm happy to remove it forever.